rusk/tests/common/state.rs

```diff
@@ -51,6 +53,7 @@ pub fn new_state<P: AsRef<Path>>(dir: P, snapshot: &Snapshot) -> Result<Rusk> {
     Ok(rusk)
 }
 
+#[allow(dead_code)]
```

Comment:
Same here on removal.

Reply:
marking this as resolved and will only track the first comment on this